cfgen/cfgen

```diff
@@ -401,6 +401,19 @@ def enrich_cluster_desc(desc: Dict[str, Any], mock_p: bool) -> None:
                 float(node['facts']['memorysize_mb']))
 
         logging.debug('Node [hostname=%s] facts=%s', host, node['facts'])
+
+        portalgroup = PortalGroup
+        if 'network_ports' not in node or not bool(node['network_ports']):
```

Comment:
This statement is unreadable. Please move this to some function to fetch network ports from CDF if not available that function must return None.

Reply:
Done